src/common/net/wifi/core/Ieee80211.cxx

```diff
@@ -11,6 +11,149 @@
 
 namespace Microsoft::Net::Wifi
 {
```

Comment:
It seems like all of the `*ToString` functions added here return the enumeration name, which is what `magic_enum::enum_name()` does. Is there any reason we can't just use that instead? If nothing else, could the functions be implemented with that instead?

The only exception is for the `Ieee80211AuthenticationAlgorithm` for which the `VendorSpecific` value exceeds `MAGIC_ENUM_RANGE_MAX`, thus preventing the use of `magic_enum::enum_name()` (and other magic_enum helpers).

Reply:
Oh good point, yeah that should work. I will remove these except for the auth algorithm one you mentioned.